samples/maps/geo-map/type-shape-polygon-series/Pages/MapTypeScatterPolygonSeries.razor

```diff
@@ -4,16 +4,23 @@
 <div class="igContainer">
     <div class="igComponent">
         <GeographicMap Height="100%" Width="100%" Zoomable="true">
```

Comment:
can you set BackgoundContent to null on the map because shape of countries do not always aligned with OpenStreetMap imagery tiles rendered by default in BackgroundContent